zScanner/ViewControllers/Documents/DocumentsListViewController.swift

```diff
@@ -87,6 +87,7 @@ class DocumentsListViewController: BaseViewController, ErrorHandling {
                     
                 case .success:
                     self.rightBarButtons = []
+                    self.documentsViewModel.lastDepartmentSelected?.select()
```

Comment:
Replace this line with `self. departmentsStackView.subvies.forEach({ ($0 as? UIButton).isSelected = false })` and remove the other changes :)

You did lot of nasty stuff in multiple files to achieve the same effect :D